fp-proj/src/app/app.component.ts

```diff
@@ -7,4 +8,22 @@ import { Component } from '@angular/core';
 })
 export class AppComponent {
   title = 'fp-proj';
+
+  loading: boolean;
+  constructor(router: Router) {
+    this.loading = false;
+    router.events.subscribe(
```

Comment:
przenioslbym to ustwaienie flagi loading do jakiegos serwisu :) i uwzywal loadera przy pobieraniu danych :) podwojny zysk